docs/libcurl/opts/CURLOPT_SSL_CTX_FUNCTION.3

```diff
@@ -72,8 +71,8 @@ See cacertinmem.c in docs/examples directory for usage example.
 
 https://curl.haxx.se/libcurl/c/cacertinmem.html
 .SH AVAILABILITY
-Added in 7.11.0 for OpenSSL. Added in 7.42.0 for wolfSSL/CyaSSL. Added in
-7.54.0 for mbedTLS. Other SSL backends not supported.
+Added in 7.11.0 for OpenSSL. Added in 7.42.0 for wolfSSL. Added in 7.54.0 for
+mbedTLS. Other SSL backends not supported.
```

Comment:
Same as above comment.

Reply:
And similarly:

Added in 7.11.0 for OpenSSL, 7.42.0 for wolfSSL and 7.54.0 for mbedTLS. Other SSL backends are not supported.